Readme.adoc

```diff
@@ -109,5 +109,34 @@ positional arguments:
 
 optional arguments:
   -h, --help            show this help message and exit
+----
+
+== Upgrading Postgres from 12 to 15
+From gateway release 1.7.3, upgrading Postgres is required. Follow the steps below to upgrade Postgres from version 12 to 15:
+
+=== Shutdown the node, and gateway:
+[source, bash]
+----
+./babylonnode docker stop
+----
 
+=== Install the new Postgres 15
```

Comment:
can we make that block multiline? Right now it requires horizontal scroll and i doubt that anyone would read it.